org.sailfishos.usermanager.xml

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.sailfishos.usermanager">
+    <method name="users">
+        <arg direction="out" type="a(ss)" name="users"/>
+        <annotation name="org.qtproject.QtDBus.QtTypeName.Out0" value="QList&lt;SailfishUser&gt;"/>
+    </method>
+    <method name="addUser">
+        <arg direction="in" type="s" name="login_name"/>
```

Comment:
I find these slightly confusing and not self-explanatory. Could they be username and (full_)name for example? Or maybe add some annotations at least?

Reply:
Switched to user_name and full_name